packages/template-ui/channel-overrides/cas/options.json

```diff
@@ -0,0 +1,5 @@
+{
+  "displayLogoInHeader": false,
+  "isEndlessApp": true,
```

Comment:
I believe `isEndlessApp` matches the `Is an Endless curated channel?` question from the checklist, correct? If so, :+1:

Reply:
Yes, exactly. That flag adds a "curated by" and the Endless logo in the channel page footer.